### ISSUE TITLE
Appbar.Action icon: improve accessibility

### ISSUE THREAD

**Issue by MBach** [callstack/react-native-paper#1344]
## Environment

Android

## Description

Default size for icon in `<Appbar.Action>` should have a greater size for accessibility. When using default size, 24, automated tests for Pre-Launch report on Google Play Console reports the following issues:

![image](https://user-images.githubusercontent.com/1532657/65394399-e7c3b200-dd8d-11e9-9f5f-1b5e9113795c.png)

I tried to change size with 48, but the result is ugly:
![image](https://user-images.githubusercontent.com/1532657/65394449-a2ec4b00-dd8e-11e9-9f0f-c7878869a17d.png)

And with some padding or margin, but without any luck. Even if it worked, we need to have a solution which works everywhere by default.

## Reproducible Demo

Any App with a11y tests passed on your personal account on GPC.

**Comment by MBach** [comment]
:exclamation:

**Comment by MBach** [comment]
What?
